SignUpPageSD.java

```diff
@@ -0,0 +1,70 @@
+package StepDefinition.Hotels;
+
+import Helper.Check;
+import Pages.Hotels.SignUpPageHotels;
+import Web.UseDriver;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+public class SignUpPageSD {
+
+    SignUpPageHotels sUp = new SignUpPageHotels();
+
+    @And("^I enter '#!@###' as email$")
+    public void typeEmail(){
+        sUp.enterEmail("#!@###");}
+
+    @And("^I enter '12345' as password$")

```

Comment:
Try to make these steps dynamic/parameterized and pass values from steps of feature file.